app/namespace.go

```diff
@@ -574,6 +574,33 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 						},
 					},
 				},
+				{
+					Name:    "retention",
+					Usage:   "The length of time a closed workflow will be preserved before deletion",
+					Aliases: []string{"r"},
+					Subcommands: []*cli.Command{{
+						Name:    "set",
+						Aliases: []string{"s"},
+						Usage:   "Set the length of time a closed workflow will be preserved before deletion per namespace",
+						Flags: []cli.Flag{
```

Comment:
shouldn't this take the ResourceVersion flag as well?

Reply:
done